doc/api/process.md

```diff
@@ -3343,6 +3342,33 @@ In custom builds from non-release versions of the source tree, only the
 `name` property may be present. The additional properties should not be
 relied upon to exist.
 
+## `process.execve(file\[, args\[, env]])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `file` {string} The name or path of the executable file to run.
+* `args` {string\[]} List of string arguments. No argument can contain a null-byte (`\u0000`).
+* `env` {Object} Environment key-value pairs.
+  No key or value can contain a null-byte (`\u0000`).
+  **Default:** `process.env`.
+
+Replaces the current process with a new process.
+
+This is achieved by using the `execve` Unix function and therefore no memory or other
```

Comment:
Is it Unix or POSIX? This makes it sounds like it would be Unix-specific thing, but we say it’s available on POSIX below

Reply:
It was meant to be POSIX, updated!